like_button.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
What's the difference between this file and the like 'src/like_button.js' file? Is one not needed anymore?